README.md

```diff
@@ -2,8 +2,7 @@
 
 [💬 Join the community on Slack](https://join.slack.com/t/thepalisadoes-dyb6419/shared_invite/zt-1s3fehykg-n2wfvgapdRxMakOQs~U7PA)
 
- 
-[![N|Solid](https://github.com/PalisadoesFoundation/talawa/blob/81c0d49458e176e8968fa42c610d220519860e06/assets/images/talawa-logo-lite-200x200.png)](https://github.com/PalisadoesFoundation/talawa) 
+![talawa-logo-lite-200x200](https://user-images.githubusercontent.com/65951872/231709493-cb09e37a-8125-4679-85eb-5d1de3b58afa.png)
```

Comment:
We should be using:
- https://raw.githubusercontent.com/PalisadoesFoundation/talawa/develop/assets/images/talawa-logo-lite-200x200.png

This is the true source of the image if you click through the repo and look at the raw source via the location https://github.com/PalisadoesFoundation/talawa/blob/develop/assets/images/talawa-logo-lite-200x200.png

Reply:
Okay @palisadoes, I'll update the path.